app/javascript/utils/index.ts

```diff
@@ -77,3 +77,72 @@ export const clickTracking = (affiliate: string, module: string, query: string,
     })
   });
 };
+
+const getLuminance = (red: number, green: number, blue: number) => {
+  const rgb = [red, green, blue].map((index) => {
+    const value = index / 255;
+    return value <= 0.03928
+      ? value / 12.92
+      : Math.pow((value + 0.055) / 1.055, 2.4);
+  });
+  return rgb[0] * 0.2126 + rgb[1] * 0.7152 + rgb[2] * 0.0722;
+};
+
+const getRgbColorObject = (color: string) => {
+  const rgbStrLen = 'rgb('.length;
```

Comment:
Can we just make this:
```suggestion
  const rgbStrLen = 4;
```
or define it as a constant elsewhere?